content/codehimanshu/drone-github-comment/index.md

```diff
@@ -0,0 +1,89 @@
+---
+version: '1.6'
+date: 2020-04-24T00:00:00+00:00
+title: Drone Github Comment
+author: codehimanshu
+tags: [ infrastructure, github, drone, comment, buildstatus ]
+logo: github.svg
+repo: gradeup/drone-github-comment
+image: codehimanshu/gitdrone
+---
+
+Use this plugin to update build status on Github Pull Request. This is useful when the complete team does not want to open drone dashboard for each build message.
+
+```yaml
+steps:
+- name: update-status-to-github
+  image: codehimanshu/gitdrone:1.6
+  pull: true
+  environment:
+    DRONE_PULL_REQUEST: ${DRONE_PULL_REQUEST}
```

Comment:
Please remove all the `DRONE_` variables, they never got to be defined like that because they are anyway always getting injected into every step.